### ISSUE TITLE
wait so we can find out all the packets that are being sent by ur client?

### ISSUE THREAD

**Issue by HausemasterIssue** [Sxmurai/packet-logger#1]
like we can look at future and pyro crits and see how far they tp up? and the same with future yawstep rotations cuz those use packets? damn. might wanna do that :)

**Comment by Sxmurai** [comment]
yeah, but i have to fix fields not showing up if they were extended (ex CPacketPlayer.Position extends CPacketPlayer, and none of the fields show up)

**Comment by HausemasterIssue** [comment]
ok, nice
